ui/react-app/web.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2023 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package reactApp
```

Comment:
For the sake of simplcity, let's call this package `reactUI` and name the file accordingly.

Reply:
nvm, as you explained offline there's benefits to this approach - downstream consumers don't have to adopt the react-ui until it's ready. I'm good with that.